src/Fixer/ClassNotation/OrderedClassElementsFixer.php

```diff
@@ -232,7 +232,7 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
             $i = $tokens->getNextTokenOfKind($i, ['{']);
             $elements = $this->getElements($tokens, $i);
 
-            if (!$elements) {
+            if (0 === \count($elements)) {
```

Comment:
```suggestion
            if ([] === $elements) {
```

Reply:
-1, I prefer to not create an array when I don't need it.